src/app/shared/form/builder/ds-dynamic-form-ui/ds-dynamic-form-control-container.component.html

```diff
@@ -20,7 +20,7 @@
             <ng-container #componentViewContainer></ng-container>
           </div>
           <small
-            *ngIf="hasHint &&
+            *ngIf="hasHint && (model.type !== 'CUSTOM_CHECKBOX') &&
```

Comment:
At least, use `DYNAMIC_FORM_CONTROL_TYPE_CUSTOM_CHECKBOX` constant instead of string value.
A better option is explained into next comment